src/statistics.cpp

```diff
@@ -758,7 +758,7 @@ config write_stats()
 
 void write_stats(config_writer &out)
 {
-	out.write_key_val("mid_scenario", mid_scenario ? "yes" : "no");
+	out.write_key_val("mid_scenario", mid_scenario ? true : false);
```

Comment:
Because `mid_scenario` is already a boolean, the ternary expression is unnecessary.